cancat/__init__.py

```diff
@@ -2194,7 +2194,7 @@ def reprCanMsgsIso(self, start_msg=0, stop_msg=None, start_baseline_msg=None, st
 
         out.append("Total Messages: %d  (repeat: %d / similar: %d)" % (msg_count, data_repeat, data_similar))
 
-        return "\n".join(out)
```

Comment:
i just fixed this last night. :)
i'm pretty sure all python complains about returning data from a yield generator.

Reply:
weird that until python 3.10 it would run this just fine.  I'll update this branch to be based on the latest from master.